client/blocks/eligibility-warnings/warning-list.tsx

```diff
@@ -1,9 +1,9 @@
 import { Card, Badge } from '@automattic/components';
-import { useHasEnTranslation } from '@automattic/i18n-utils';
 import { localize, LocalizeProps, translate } from 'i18n-calypso';
 import { Fragment } from 'react';
 import ActionPanelLink from 'calypso/components/action-panel/link';
 import ExternalLink from 'calypso/components/external-link';
+import { preventWidows } from 'calypso/lib/formatting';
```

Comment:
I didn't realize we had/used this formatting function. I think we generally handle them with css now.